### ISSUE TITLE
Пресеты миниатюр не соответствуют своим большим изображениям

### ISSUE THREAD

**Issue by Yar-rus** [instantsoft/icms2#856]
**Используемая версия InstantCMS**: 2.10.0

**Как воспроизвести проблему**:

В ТК "статьи" на демо сайте создаю поле photo;
Тип: Набор изображений;
Размер в списке: Средний;
Размер в записи: Большой;
Пресет миниатюр: Маленький;
Включаем "Выделять первое изображение в записи";
Максимальное количество изображений: 17;
Группа: Фото;
Показывать в записи;
Показывать в списке;
Положение заголовков: не показывать нигде;
Загружаю фотографии, сохраняю;
Затем включаю режим редакции, удаляю все фото, сохраняю; 
Включаю редакцию, загружаю снова фото.

В итоге все большие фото разные, как и положено, а вот некоторые превью не соответствуют, отображаются превью от другого изображения.

**Воспроизводится ли проблема на демо http://demo.instantcms.ru/**: ДА

Как картинку загрузить сюда не знаю, поэтому размещаю на форуме: http://www.instantcms.ru/forum/thread29846-1.html#291575

**Comment by fuzegit** [comment]
Либо дайте еще информации о проблеме, либо мне придётся закрыть этот тикет, т.к. не могу воспроизвести проблему. Может вы переставляли порядок фото или еще что-то делали.

**Comment by Yar-rus** [comment]
Да, порядок менял. Сейчас попробовал снова на демосайте. Добавлял, удалял и снова добавлял, вроде нормально. Потом поменял порядок, удалил и снова загрузил, тут ошибка и появилась снова, действительно.
